examples/js/postprocessing/SAOPass.js

```diff
@@ -74,7 +74,12 @@ THREE.SAOPass = function ( scene, camera, depthTexture, useNormals, resolution )
 
 	}
 
-	this.saoMaterial = new THREE.ShaderMaterial( THREE.SAOShader );
+	this.saoMaterial = new THREE.ShaderMaterial( {
+		defines: THREE.SAOShader.defines,
```

Comment:
As i said in https://github.com/mrdoob/three.js/issues/13370#issuecomment-367394021, it is also necessary to clone the `defines` object because the respective values can differ from pass to pass. If you would use different camera types (perspective vs. orthographic) in your two scenes, it would be a problem. That should do it
```js
defines: Object.assign( {}, THREE.SAOShader.defines ),
```

Reply:
oh, thought your code was already the complete one^^

Reply:
inserted the line of code ;)